main.py

```diff
@@ -0,0 +1,100 @@
+"""
+Модуль, який містить класи Fish і Aquarium.
+"""
+
+class Fish:
+    """
+    Клас, що представляє рибу.
+    """
+
+    def __init__(self, name, age, species, *args):
+        self.name = name
+        self.age = age
+        self.species = species
+        self.size = args[0]
+        self.preferred_food = args[1]
+        self.is_aggressive = args[2]
+        self.needed_space = args[3]
+
+    def __str__(self):
+        return f"Рибка {self.name} {self.age} років"
+
+    def __repr__(self) -> str:
+        pass
+
+class Aquarium:
+    """
+    Клас, що представляє акваріум.
+    """
+
+    def __init__(self, total_volume):
+        self.total_volume = total_volume
+        self.free_space = total_volume
+        self.aggressive_fish_list = []  # Окремий список для агресивних риб
+        self.non_aggressive_fish_list = []  # Окремий список для неагресивних риб
+
+    def add_fish(self, fish):
+        """
+        Додає рибу до відповідного списку акваріума.
+        """
+        if fish.is_aggressive:
+            # Перевірка, чи є вільне місце для агресивної риби
+            if self.free_space >= fish.needed_space:
+                self.aggressive_fish_list.append(fish)
+                self.free_space -= fish.needed_space
+                print(f"{fish.name} (агресивна) додана до акваріума для агресивних риб.")
+            else:
+                print(f"Недостатньо місця для {fish.name} (агресивна) в акваріумі.")
+        else:
+            # Перевірка, чи є вільне місце для неагресивної риби
+            if self.free_space >= fish.needed_space:
+                self.non_aggressive_fish_list.append(fish)
+                self.free_space -= fish.needed_space
+                print(f"{fish.name} додана до акваріума для неагресивних риб.")
+            else:
+                print(f"Недостатньо місця для {fish.name} в акваріумі.")
+
+    def get_top_3_fish(self, fish_list):
```

Comment:
get_top_3_fish - by what exactly? I have no idea what is the criteria to select fish without reading the code